apps/src/sites/studio/pages/certificates/show.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
 import getScriptData from '@cdo/apps/util/getScriptData';
 import CertificateShare from '@cdo/apps/templates/CertificateShare';
+import {getStore} from '@cdo/apps/redux';
 
 $(document).ready(function() {
+  const store = getStore();
   const certificateData = getScriptData('certificate');
-  const {imageUrl, printUrl} = certificateData;
+  const {imageUrl, printUrl, announcement} = certificateData;
   ReactDOM.render(
-    <CertificateShare imageUrl={imageUrl} printUrl={printUrl} />,
+    <Provider store={store}>
```

Comment:
Just making sure I understand -- we're connecting to Redux here so that `TwoColumnActionBlock` can get the `responsiveSize`, right?

Reply:
correct!